saleor/dashboard/discount/forms.py

```diff
@@ -21,6 +21,22 @@ class SaleForm(forms.ModelForm):
     class Meta:
         model = Sale
         exclude = []
+        labels = {
+            'name': pgettext_lazy(
+                'Sale form label',
+                'Sale name'),
```

Comment:
You could omit the ```Sale``` keyword in message. Same below.

Reply:
Sure, changed here and in other places to make naming more consistent.